### PR TITLE
Add information about OpenHPC based containers

### PR DESCRIPTION
**Pull request by adrianreber**
(This is still using my private quay.io registry URLs, which can be replaced later with the real location)

During SC19 there were a couple of questions about OpenHPC based containers. Although it is easy to create an OpenHPC based container (see Dockerfiles) it is even more convenient if it already exists.

Details about the containers can be found in the README.md file.

 * `make` # to build all container
 * `make push` # build and push all container to a registry
 * `make check` # run some simple checks using the containers

### CODE REVIEW

**Comment by adrianreber** [comment]
@koomie @crbaird As discussed in Denver. This is a try to provide OpenHPC based containers.

File: containers/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# OpenHPC container images
+
+> (Everything defaults to `podman` as it provides the possibility to built
```
Comment (boegel) [inline containers/README.md]: `built` should be `build` here?

Comment (adrianreber) [reply containers/README.md]: Thanks. Fixed and force pushed!

**Comment by koomie** [comment]
I landed this on the 2.0 branch. These will presumably need to get updated to use the `gnu9` or `gnu10` variant depending on what version ships with 2.0.  Now that there is a containers subdirectory, I moved the previous singularity example to reside here as well.

**Comment by adrianreber** [comment]
> I landed this on the 2.0 branch. These will presumably need to get updated to use the `gnu9` or `gnu10` variant depending on what version ships with 2.0. Now that there is a containers subdirectory, I moved the previous singularity example to reside here as well.

@koomie Just let me know whenever I need to adapt/change anything with this.

**Pull request closed without merging**